### ISSUE TITLE
You can softlock yourself by hitting lock

### ISSUE THREAD

**Issue by dominichayesferen** [Vanilla-OS/vanilla-installer#257]
You can press lock in the quick actions area and softlock yourself as there's no password set for the user.

**Comment by taukakao** [comment]
Maybe adding
org.gnome.desktop.lockdown disable-lock-screen true
to https://github.com/Vanilla-OS/live-iso/blob/orchid/etc/config/hooks/live/080-disable-gnome-lock.chroot
I will try that out soon.

**Comment by taukakao** [comment]
Yep, that seems to fix it, though I'm having a lot of other bugs now. 
I have to check if those are caused by the change or not.
